core/trino-main/src/test/java/io/trino/connector/MockConnector.java

```diff
@@ -144,7 +146,8 @@
             BiFunction<ConnectorSession, ConnectorTableHandle, ConnectorTableProperties> getTableProperties,
             Supplier<Iterable<EventListener>> eventListeners,
             MockConnectorFactory.ListRoleGrants roleGrants,
-            MockConnectorAccessControl accessControl, Function<SchemaTableName, List<List<?>>> data)
+            MockConnectorAccessControl accessControl, Function<SchemaTableName, List<List<?>>> data,
```

Comment:
please put each argument into separate line. (Actually it was my fault, I put `data` into the same line).